pkg/kubelet/network/cni/cni.go

```diff
@@ -43,6 +44,8 @@ type cniNetworkPlugin struct {
 	host           network.Host
 	execer         utilexec.Interface
 	nsenterPath    string
+	mu             sync.Mutex
+	podMutexes     map[string]*sync.Mutex
```

Comment:
Yes, but the issue I mentioned may take sometime to fix. 

I guess I don't fully understand the impact of this bug in its current status. If the pod object has already been removed by the apiserver (which is the cause of the race condition), is the error returned by cni going to cause any damage?